opm/simulators/linalg/PreconditionerFactory.hpp

```diff
@@ -143,12 +143,33 @@ class PreconditionerFactory
         using SmootherArgs = typename Dune::Amg::SmootherTraits<Smoother>::Arguments;
         SmootherArgs smootherArgs;
         smootherArgs.iterations = prm.get<int>("iterations", 1);
+	//int iluwitdh = prm.get<int>("iluwidth", 0);
+	//MILU_VARIANT milu = convertString2Milu(prm.get<std::string>("milutype",std::string("ilu")));
+	//smootherArgs.setMilu(milu);
+	//smootherArgs.setN(iluwitdh);
         // smootherArgs.overlap=SmootherArgs::vertex;
         // smootherArgs.overlap=SmootherArgs::none;
         // smootherArgs.overlap=SmootherArgs::aggregate;
-        smootherArgs.relaxationFactor = prm.get<double>("relaxation", 0.9);
+        smootherArgs.relaxationFactor = prm.get<double>("relaxation", 1.0);
```

Comment:
Maybe this deserves to be mentioned in the commit message.

Reply:
Do you have an opinion? The default settings for flow_blackoil_dunecpr is anyway 1.0.